app/frontend/packs/nationalities-component.js

```diff
@@ -30,31 +30,33 @@ const nationalitiesComponent = () => {
   hideSection(thirdSelectEl, thirdFormLabel);
 
   function addRemoveLink(labelEl, selectEl) {
+    const parentEl = labelEl.parentElement;
+
     const removeLink = document.createElement("a");
     removeLink.innerHTML = "Remove";
     removeLink.classList.add("govuk-link", "app-nationality__remove-link");
 
     // This has to be a link and not a button as the govuk-link class requires an
-    // href to apply  it's styling
+    // href to apply it’s styling
     removeLink.href = "#";
-    labelEl.appendChild(removeLink);
+    parentEl.insertBefore(removeLink, labelEl);
```

Comment:
Was this change the result of an accessibility issue? If so might be worth adding to the commit msg ...